src/CONST.ts

```diff
@@ -1463,6 +1463,12 @@ const CONST = {
     },
     QUICKBOOKS_ONLINE: 'quickbooksOnline',
 
+    QUICKBOOKS_DESKTOP_CONFIG: {
+        MARK_CHECKS_TO_BE_PRINTED: 'markChecksToBePrinted',
+        REIMBURSABLE_EXPENSES_ACCOUNT: 'reimbursableExpensesAccount',
```

Comment:
You took the same keys as QBO for QBD, but that's not the case. These keys don't exist in QBD.

Please see the connection's data sample that was [here](https://docs.google.com/document/d/1LQNlTBlHjFqRdru_4UkqVr0QRseNNDq2IhFFpXcgfwY/edit?tab=t.0#bookmark=id.pivvkql7f8zz).

It should be in `export.reimbursableAccount`